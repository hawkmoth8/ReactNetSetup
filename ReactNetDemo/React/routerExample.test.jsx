import RouterExample from "./routerExample";

describe("router example tests", () => {
  it("should render", () => {
    const wrapper = shallow(<RouterExample />);
    expect(wrapper).toMatchSnapshot();
  });

  it("should fail", () => {
    expect(1).toBe(1);
  });
});
