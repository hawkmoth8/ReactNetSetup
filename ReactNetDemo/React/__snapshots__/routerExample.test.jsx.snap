// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`router example tests should render 1`] = `
<BrowserRouter>
  <ul>
    <li>
      <Link
        to="/privacy"
      >
        privacy
      </Link>
    </li>
    <li>
      <Link
        to="/page1"
      >
        page1
      </Link>
    </li>
    <li>
      <Link
        to="/page2"
      >
        page2
      </Link>
    </li>
  </ul>
  <Switch>
    <Route
      exact={true}
      path="/privacy"
    >
      <div>
        this is privacy
      </div>
    </Route>
    <Route
      exact={true}
      path="/page1"
    >
      <div>
        this is page1
      </div>
    </Route>
    <Route
      exact={true}
      path="/page2"
    >
      <div>
        this is page2
      </div>
    </Route>
  </Switch>
</BrowserRouter>
`;
